### PR TITLE
Fix nightly runs

### PR DESCRIPTION
**Pull request by ashwini-mhatre**
##### ISSUE TYPE

- Bugfix Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: test-requirements.txt [hunk 1]
```diff
@@ -12,4 +12,4 @@ pytest-xdist
 pexpect
 git+https://github.com/ansible-network/pytest-ansible-network-integration.git
```
Comment (NilashishC) [inline test-requirements.txt:13]: Unrelated, but can you please fix the pytest-ansible requirements in this collection as well?

File: test-requirements.txt [hunk 2]
```diff
@@ -4,12 +4,11 @@ flake8
...
 
 # Unit test runner
-pytest-ansible ; python_version >= '3.9'
-git+https://github.com/ansible-community/pytest-ansible-units.git ; python_version < '3.9'
+pytest-ansible
 pytest-xdist
 
 # For integration tests
 pexpect
 git+https://github.com/ansible-network/pytest-ansible-network-integration.git
```
Comment (NilashishC) [inline test-requirements.txt:12]: Same as above.

**Comment by doc-sheet** [comment]
Hi! I suppose it could be better to just pin to `<=1.0.0` because in some environments netaddr is way older then 0.10.1

**Review by NilashishC — CHANGES_REQUESTED** [review]
Current changes look good. But seems like new lint failures have shown up. We'll need to fix that.

**Review by NilashishC — APPROVED** [review]

**Pull request merged**